test/libsolidity/smtCheckerTests/modifiers/modifier_two_invocations_2.sol

```diff
@@ -0,0 +1,19 @@
+pragma experimental SMTChecker;
+
+contract C
+{
+	uint x;
+
+	modifier m {
+		require(x == 0);
+		_;
+		x = x + 1;
+		assert(x <= 2);
+	}
+
+	function f() m m public {
```

Comment:
Can you also add another similar test but with `f() m m m m` so it should fail 3 times?

Reply:
So, I tried that an it currently fails only once, not three times.
The reason for that is that even if it creates 4 verification targets (for the assertion in 4 modifiers), after detecting the first violation, the target is recorded as unsafe, but it is recorded per ASTNode, which is actually the same for all these 4 targets. So the other checks are skipped. If we want to report 3 failures here, then we need to change how we store the safe/unsafe targets after they have been checked.

Reply:
Ah right, let's leave it like this then